web/app/js/components/Tap.jsx

```diff
@@ -22,11 +22,14 @@ class Tap extends React.Component {
   constructor(props) {
     super(props);
     this.api = this.props.api;
+    this.tapResultsById = {};
+    this.tapResultFilterOptions = this.getInitialTapFilterOptions();
+    this.debouncedWebsocketRecvHandler = _.throttle(this.updateTapResults, 500);
```

Comment:
I am wondering if this should be renamed to `debouncedWebsocketRecvHandler` instead. I realize that this is still called `debounced...` in TopModule.jsx so it's probably fine to keep it as it is.